posthog/migrations/0391_organization_is_hipaa_and_more.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.1.13 on 2024-02-20 00:06
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0390_personalapikey_scopes"),
+    ]
+
+    operations = [
```

Comment:
Am going to leave this hipaa change here and move the "trusted_customer_scores" migration into a separate PR